app/views/shipit/deploys/_deploy.html.erb

```diff
@@ -21,6 +21,7 @@
         <span class="code-additions">+<%= deploy.additions %></span>
         <span class="code-deletions">-<%= deploy.deletions %></span>
         <% if deploy.ignored_safeties? %><span class="ignored-safeties">ignoring safeties</span><% end %>
+        <% if (deploy.aborting? || deploy.aborted?) && deploy.aborted_by %><span class="aborted-by">aborted by <%= deploy.aborted_by&.login %></span><% end %>
```

Comment:
I only added to the deploy partial, should we put in all the tasks partials?

Reply:
Not a big deal. If the information is available is the important part.

Reply:
Is the `&.login` necessary? Since the `if` already check for it's presence.

Also I'd remove the `aborting? || aborted?` check, since post abort it could transition to `flapping` state or something like that.

Reply:
Oh and the `&.` makes the build fail since Shipit doesn't require 2.3 yet.

Reply:
Fixed